syspurpose/src/syspurpose/utils.py

```diff
@@ -89,3 +90,23 @@ def in_container():
     if os.path.exists(HOST_CONFIG_DIR):
         return True
     return False
+
+def make_utf8(obj):
+    """
+    Transforms the provided string into unicode if it is not already
+    :param obj: the string to decode
+    :return: the unicode format of the string
+    """
+    if is_python_version_3():
+        return obj
+    elif obj is not None and not isinstance(obj, unicode):
+        obj =  obj.decode('utf-8')
+        return obj
+    else:
+        return obj
+
+def is_python_version_3():
```

Comment:
Is syspurpose going to require `six`?  I thought it was.  If so I think we could use `if six.PY3`